### DISCUSSION TITLE
Syncing problem ipad / iphone

### DISCUSSION THREAD

**Discussion by MicheleDellaquila** [streetwriters/notesnook#5678] [Q&A]
Hi, I signed up to notesnook this day and I dowloaded desktop, mobile and ipad app but I have a problem with the last 2. When I have logged into mobile or tablet I have had a problem with sync of my notes. This message is: "User encryption key not generated, please relogin". I do that but nothing, help me pls

**Comment by ammarahm-ed** [comment]
Hi this issue has been fixed in v3.0.7 of mobile and ipad apps. Please update to fix the issue.
